pkgs/by-name/ra/radiance/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  libGL,
+  libGLU,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "radiance";
+  version = "5.4";
+
+  src = fetchFromGitHub {
+    owner = "LBNL-ETA";
+    repo = pname;
+    rev = "3be7601c";
+    hash = "sha256-CRf6vyVL7RpUi1V3VTTYknGzC/BlrACkDhyDG04sg7I=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    libGL
+    libGLU
+  ];
+
+  meta = with lib; {
+    description = "A Validated Lighting Simulation Tool";
+    homepage = "https://github.com/LBNL-ETA/Radiance";
+    license = licenses.radiance;
+    maintainers = with maintainers; [ robwalt ];
+    mainProgram = "rad";
+  };
+}
```

Close the parenthesis opened above before `finalAttrs`
```suggestion
})
```